app/controllers/group_assignment_invitations_controller.rb

```diff
@@ -124,16 +140,43 @@ def organization
   end
   helper_method :organization
 
+  def classroom_config
+    starter_code_repo_id = group_assignment_repo.starter_code_repo_id
+
+    return nil unless starter_code_repo_id
```

Comment:
you can leave out the `nil`, `return unless starter_code_repo_id` will return `nil`.